workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -29,45 +51,81 @@ export const ChangelogHistory: FC = () => {
           </div>
 
           <ol className={classes.commitsList}>
-            {batchCommits.map((batchCommit, i) => (
-              <li className={classes.commitsListItem} key={batchCommit.batchId}>
-                <div className={classes.commitDetails}>
-                  <h4 className={classes.commitMessage}>
-                    {batchCommit.commitMessage}
-                  </h4>
-                  <span className={classes.commitTime}>
-                    {formatTimeAgo(new Date(batchCommit.createdAt))}
-                  </span>
-                </div>
-
-                <div className={classes.commitControls}>
-                  <Button
-                    className={classes.commitCompareButton}
-                    href={
-                      i === 0
-                        ? documentationPage.linkTo()
-                        : changelogPage.linkTo(batchCommit.batchId)
-                    }
-                  >
-                    Compare
-                  </Button>
-
-                  {i !== 0 && (
-                    <Button
-                      className={classes.commitResetButton}
-                      onClick={() => {
-                        console.log(`resetting to ${batchCommit.batchId}`);
-                      }}
-                    >
-                      Reset
-                    </Button>
-                  )}
-                </div>
-              </li>
-            ))}
+            {batchCommits.map((batchCommit, i) => {
```

Comment:
At some point we could also move this into it's own component (such that changes in render don't rerender the entire list) - unlikely to happen in this case (editing commit messages, etc)